libcxx/include/ios

```diff
@@ -590,7 +615,13 @@ protected:
 
 private:
   basic_ostream<char_type, traits_type>* __tie_;
-  mutable int_type __fill_;
+
+#if defined(_LIBCXX_IOS_USE_FILL_HELPER)
+  using _FillType = _FillHelper<_Traits>;
+#else
+  using _FillType = _SentinelValueFill<_Traits>;
+#endif
+  mutable _FillType __fill_;
```

Comment:
minor nit: I'd suggest `traits_type` instead of `_Traits` as the parameter here for consistency which the style of the rest of the class definition:
```suggestion
#if defined(_LIBCXX_IOS_USE_FILL_HELPER)
  using _FillType = _FillHelper<traits_type>;
#else
  using _FillType = _SentinelValueFill<traits_type>;
#endif
  mutable _FillType __fill_;
```

Reply:
Good catch, thanks!